### ISSUE TITLE
Symbol not found find_last_not_of

### ISSUE THREAD

**Issue by jaraco** [kitao/pyxel#302]
I believe I've encountered the issue reported in #257.

Using Python 3.9.2 as installed from Python.org on MacOS 11.2.3, I've used `brew install sdl2 sdl2_image gifsicle` and then install pyxel 1.4.3 from wheel, then attempt to install the examples, but it fails with this error:

```
draft $ install_pyxel_examples
Traceback (most recent call last):
  File "/Users/jaraco/.local/bin/install_pyxel_examples", line 5, in <module>
    from pyxel.examples import install
  File "/Users/jaraco/.local/pipx/venvs/pyxel/lib/python3.9/site-packages/pyxel/__init__.py", line 10, in <module>
    from . import core  # type: ignore
  File "/Users/jaraco/.local/pipx/venvs/pyxel/lib/python3.9/site-packages/pyxel/core/__init__.py", line 42, in <module>
    _lib = _load_library()
  File "/Users/jaraco/.local/pipx/venvs/pyxel/lib/python3.9/site-packages/pyxel/core/__init__.py", line 39, in _load_library
    return cdll.LoadLibrary(lib_path)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/ctypes/__init__.py", line 452, in LoadLibrary
    return self._dlltype(name)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/ctypes/__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: dlopen(/Users/jaraco/.local/pipx/venvs/pyxel/lib/python3.9/site-packages/pyxel/core/bin/macos/libpyxelcore.dylib, 6): Symbol not found: __ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE16find_last_not_ofEPKcmm
  Referenced from: /Users/jaraco/.local/pipx/venvs/pyxel/lib/python3.9/site-packages/pyxel/core/bin/macos/libpyxelcore.dylib
  Expected in: /usr/lib/libstdc++.6.dylib
 in /Users/jaraco/.local/pipx/venvs/pyxel/lib/python3.9/site-packages/pyxel/core/bin/macos/libpyxelcore.dylib
```

**Comment by jaraco** [comment]
Aha. If I pass `--no-binary pyxel` to the pip install (build from source), the install now works.

**Comment by keigo0205** [comment]
I have the same problem.

I solved this problem using the method below.
```
python -m pip uninstall pyxel
python -m pip install --no-binary :all: pyxel
```

thanks!!

**Comment by mogocat** [comment]
Same issues here and thanks for the solutions!

**Comment by ryohji** [comment]
To solve this problem fundumentally, it seems that we need to make `libpyxelcore.dylib` from source.

1. Clone source from github. `git clone https://github.com/kitao/pyxel.git`
2. Edit `pyxel/pyxel/core/Makefile` line 28, from `CC = g++-10` to `CC = g++` (or `CC = c++` if you want to use Xcode CLT).
3. Delete problematic dylib. `rm pyxel/pyxel/core/bin/macos/libpyxelcore.dylib`
4. Make new dylib. `make -C pyxel/pyxel/core`
5. Install package from local copy. `python3 -m pip install --use-feature=in-tree-build ./pyxel`

----
The error message "Symbol not found" saids that the symbol referenced by libpyxelcore.dylib is not found in our system libraries.
The libpyxcelcore.dylib included in current pyxel 1.4.3 package was built using g++-10, but for newer macOS it replaced by g++-11 and its libc++.dylib do not have the symbol. So we need make fresh dylib according to our system.

**Comment by the-mba** [comment]
> To solve this problem fundumentally, it seems that we need to make `libpyxelcore.dylib` from source.
> 
> 1. Clone source from github. `git clone https://github.com/kitao/pyxel.git`
> 2. Edit `pyxel/pyxel/core/Makefile` line 28, from `CC = g++-10` to `CC = g++` (or `CC = c++` if you want to use Xcode CLT).
> 3. Delete problematic dylib. `rm pyxel/pyxel/core/bin/macos/libpyxelcore.dylib`
> 4. Make new dylib. `make -C pyxel/pyxel/core`
> 5. Install package from local copy. `python3 -m pip install --use-feature=in-tree-build ./pyxel`
> 
> The error message "Symbol not found" saids that the symbol referenced by libpyxelcore.dylib is not found in our system libraries. The libpyxcelcore.dylib included in current pyxel 1.4.3 package was built using g++-10, but for newer macOS it replaced by g++-11 and its libc++.dylib do not have the symbol. So we need make fresh dylib according to our system.

I don't know if people do replies in github but, man, thank you so much!!! You are the best, I was about to open VirtualBox and install a fucking Windows. THANK YOU SO MUCH!!

**Comment by kitao** [comment]
Pyxel 1.5.0 is re-implemented in Rust. And Mac version doesn't depend on external modules any more.
